core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -1685,10 +1702,9 @@ object ReassignPartitionsCommand extends Logging {
       opts.cancelOpt -> collection.immutable.Seq(
         opts.reassignmentJsonFileOpt
       ),
-      opts.listOpt -> collection.immutable.Seq(
-      )
+      opts.listOpt -> collection.immutable.Seq.empty
```

Comment:
seems to break the symmetry a bit, doesn't it?  Although, I don't feel strongly about this.

Reply:
I can revert. It looked a little more reasonable when next to the suggested alter command.